src/core/models/dynamicfilters.js

```diff
@@ -39,7 +39,7 @@ export default class DynamicFilters {
 
     return new DynamicFilters({
       filters: dynamicFilters,
-      resultsContext: resultsContext
+      resultsContext: response.resultsContext
```

Comment:
can you access an array with a dot modifier?

Reply:
oh response isn't an array! it's an object

Reply:
response.facets is the array

Reply:
im silly

Reply:
does it matter if resultsContext is empty?

Reply:
is fine in this case, but it should never be empty! the _parseVerticalResponse method in searchdatatransformer.js always assigns a resultsContext. I wanted to avoid adding defaults for results context everywhere it's used if possible.